Nexd/Nexd/ui/helper/CheckoutViewController.swift

```diff
@@ -39,6 +39,15 @@ class CheckoutViewController: ViewController<CheckoutViewController.ViewModel> {
                 viewModel.navigator.toDeliveryConfirmationScreen(helpList: viewModel.helpList)
             }
         }
+
+        let backButtonTitle = Driver.just(R.string.localizable.back_button_title().asNegativeButtonText())
```

Comment:
I was wondering that probably we should introduce a new style for the back button (e.g. `.asBackButtonText()`) and make sure it implements the correct design for these back buttons! (you can find our design here: https://www.figma.com/file/fUARNhS4bLEgXZhbdm64Tk/Nexd-(Copy)?node-id=297%3A6)

- so that style would then use:
  - Proxima Nova Soft Bold font
  - and `darkButtonText` text color...

Reply:
I will adapt my code to match the design. Currently the chevron_left icon is only available in red. I saw usage of SwiftUI, so its save to assume that the target is iOS 13+? This would create the possibility to use SF Symbols which is natively integrated. 
SF Symbols chevron.left looks the same and is tintable.

Reply:
Yes, target is iOS 13+. ..

an alternative to using SF Symbols is to use the `.withTintColor` function fo UIImage... thats what I usually did in the rest of this project:

```
button.image = R.image.chevron()?.withTintColor(style.borderColor!)
```

Reply:
@KevinPeplinski: If you want to join the "Nexd" team we could also invite you to our slack channel. Just let me know and we'll send out an invite to you...

Reply:
oh, and another thing regarding the back buttons: yesterday I merged some new screens (`Transcribe*View.swift` - implemented in SwiftUI) ... we could also add the back buttons to these screens as well..

Reply:
@schroepf added a new commit including the back button for SwiftUI